go.mod

```diff
@@ -9,9 +9,9 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/rhysd/actionlint v1.6.24
 	go.uber.org/multierr v1.11.0
-	go.viam.com/api v0.1.331
-	go.viam.com/rdk v0.38.1-0.20240822173850-bc4acc1d0071
-	go.viam.com/utils v0.1.94
+	go.viam.com/api v0.1.336
```

Comment:
I don't know why this line got updated. Maybe it's for compatibility with the new version of the RDK library? I'm not purposely changing any uses of `api`; I'm going to try setting it back to its previous value.

Reply:
good point: I'll remove the last commit so it's updated after all.